src/Sylius/Bundle/ShopBundle/EventListener/AlreadyLoggedInUserRegistrationListener.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\EventListener;
+
+use Sylius\Bundle\ResourceBundle\Event\ResourceControllerEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+
+final class AlreadyLoggedInUserRegistrationListener implements EventSubscriberInterface
```

Comment:
```suggestion
final class AlreadyLoggedInUserRegistrationSubscriber implements EventSubscriberInterface
```

According to existing codebase naming scheme, it's either should be renamed to `AlreadyLoggedInUserRegistrationSubscriber` or re-done as listener

![image](https://github.com/Sylius/Sylius/assets/870747/11e529ce-7c19-4704-be0b-e1e5f86282d9)

@lchrusciel hi, what do you think adding `php-arkitect` rule to enforce this?

Reply:
My bad, I declared both as subscriber and listener inside service. I will stick as listener